desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -20,6 +24,35 @@ TabDiveInformation::TabDiveInformation(QWidget *parent) : TabBase(parent), ui(ne
 	QStringList atmPressTypes { "mbar", get_depth_unit() ,"use dc"};
 	ui->atmPressType->insertItems(0, atmPressTypes);
 	pressTypeIndex = 0;
+	// This needs to be the same order as enum dive_comp_type in dive.h!
+	QStringList types = QStringList();
+	for (int i = 0; i < NUM_DIVEMODE; i++)
+		types.append(gettextFromC::tr(divemode_text_ui[i]));
+	ui->diveType->insertItems(0, types);
+	connect(ui->diveType, SIGNAL(currentIndexChanged(int)), this, SLOT(diveMode_Changed(int)));
+	ui->scrollAreaWidgetContents_3->setStyleSheet("QGroupBox { border: 1px solid #c0c0c0; border-radius: 4px; margin-top: 0.65em; background-color: #e4e4e4;} QGroupBox::title { color: #0000c0;} ");
```

Comment:
`#c0c0c0` is `silver` if I'm not mistaken - so `border: 1px solid silver` would be nicer
similarly, `#e4e4e4` is `whisper` (don't ask me why)
`#0000c0` has no name (`#0000cd` would be `medium blue`)

Reply:
Done.